dotnet-datetime/CheckIfDateIsLessThanOrEqualToToday/CheckIfDateIsLessThanOrEqualToToday/CheckDateMethods.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Globalization;
+
+namespace CheckIfDateIsLessThanOrEqualToToday;
+
+public class CheckDateMethods
```

Comment:
@cjaynjoku 
We should rename this class as it currently reads like a method. We could rename as `DateComparison`, `DateCheck` or something different.
This naming will also be propagated across other class names that reads like methods